src/hooks/useLoadUpdateDealDeps.ts

```diff
@@ -138,8 +138,10 @@ const useLoadUpdateDealDeps: UseLoadUpdateDealDeps = (dealId) => {
             priorities,
             accountInfo,
         ].every(({ isLoading }) => Boolean(isLoading)),
-        deal: get(deal, ["data", "properties"], {}) || {},
+        deal: get(deal, ["data", "properties"]) as Deal["properties"],
         pipelines: get(pipelines, ["data", "results"], []) || [],
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
```

Comment:
Why do we need to introduce adding `// @ts-ignore` what issue is it hiding/supressing and can we fix it by ensuring types are accurate instead?

Reply:
fixed